docs/getting_started.md

```diff
@@ -10,6 +10,12 @@ CentOS 7 requires a more modern gcc to build Linux. If you receive an error such
 
 ### Ubuntu (18.04 or 20.04)
 
+For Ubuntu, we recommend updating existing pacakges before installing the prerequisites.
```

Comment:
*packages.  

This is a pretty dangerous recommendation to make in the general case.  If anything, we should only say to upgrade the dependencies themselves. I'd prefer to leave off this line entirely though

Reply:
removing